simple/services-kubernetes.yml

```diff
@@ -86,6 +88,8 @@ spec:
             value: "1"
           - name: JAEGER_REPORTER_LOG_SPANS
             value: "true"
+          - name: JAEGER_TAGS
```

Comment:
The `withConstLabel` is to add version to the metric labels (i.e. prometheus), but this `JAEGER_TAGS` environment variable is to set the `version` as a tag on all span data - so one is for metrics and the other is for the trace data.

Originally I was going to get the metric label configured to pull the version from the span tags - but that would still have required configuration on the metrics reporter, but using `withTagLabel` instead of `withConstLabel`.